### ISSUE TITLE
[Feature Request] Freshchat support

### ISSUE THREAD

**Issue by coliff** [calibreapp/react-live-chat-loader#109]
## What problem would this feature solve?

Adding Freshdesk's Freshchat (https://www.freshworks.com/live-chat-software/) widget to a webpage adds a huge amount of additional JavaScript. 

## Describe the solution you’d like to see
I'd like to see react-live-chat-loader add support for Freshchat

**Comment by plusplushq** [comment]
Hello,
@coliff @benschwarz
I'm working on improving the performance and Core Web Vitals of some of our apps, and also see that Freshdesk chat is adding a huge amount of Javascript and effecting Lighthouse scores. 
I would be interested in working with you all on this too if I can help.

**Comment by coliff** [comment]
heya @plusplushq and @benschwarz - sorry I didn't follow up on this... I realised I ideally needed a native JS solution for the Freshchat loader and not a React one so I made one:
https://github.com/coliff/freshdesk-messaging-facade#readme

It's less than 1 KB gzipped - 1/1000th of the size of the Freshdesk JavaScript. Maybe you might find it helpful. If you do work on integrating Freshdesk into the React Live Chat Loader though that could be great - and I might be able to help with any testing.

**Comment by benschwarz** [comment]
Awesome @coliff 😄 

I don't have the bandwidth to add Freshchat to react-live-chat-loader, but if there's someone out there who can help with that, we'll gladly review a PR!

**Comment by motiko** [comment]
Hey @coliff
If you still interested in this I might help. Initial problem I see with this chat bot it embeds in open state. Is there a chat icon that you can propose we can use to activate the chat bot?

**Comment by coliff** [comment]
Hiya @motiko - I don't use React so instead of working on integrating Freshdesk Messaging with react-live-chat-loader I created a simple standalone instead: https://github.com/coliff/freshdesk-messaging-facade/. For the icon I used the same one that they use set as an [SVG background image](https://github.com/coliff/freshdesk-messaging-facade/blob/main/css/freshdesk-messaging-facade.css#L20).

**Comment by motiko** [comment]
Nice implementation. In that case I would only pick it up if there is some interest. I see few stale PRs is this still actively maintained?

**Comment by coliff** [comment]
Yes, I think it's actively maintained - CHANGELOG looks busy :-)
https://github.com/calibreapp/react-live-chat-loader/blob/main/CHANGELOG.md
